<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_screen</class>
 <widget class="QWidget" name="player_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="repeat_pb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pb">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="volume_l">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_l">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_l">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QSlider" name="songProgress_s">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="songProgress_l">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration/Fulltime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_pb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_pb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>228</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_pb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>228</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QSlider" name="volume_s">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>22</width>
     <height>156</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="shuffle_pb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QLabel" name="playlist_l">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Playlist Title</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
